codespell_lib/tests/test_basic.py

```diff
@@ -100,15 +100,15 @@ def test_basic(tmpdir, capsys):
     os.remove(fname)
 
     with open(op.join(d, 'bad.txt'), 'w') as f:
-        f.write('abandonned\nAbandonned\nABANDONNED\nAbAnDoNnEd')
-    assert cs.main(d) == 4
+        f.write('abandonned\nAbandonned\nABANDONNED\nAbAnDoNnEd\nabandonned\rAbandonned\r\nABANDONNED \n AbAnDoNnEd')  # noqa: E501
+    assert cs.main(d) == 8
     code, _, stderr = cs.main('-w', d, std=True)
     assert code == 0
     assert 'FIXED:' in stderr
-    with open(op.join(d, 'bad.txt')) as f:
+    with open(op.join(d, 'bad.txt'), newline='') as f:
```

Comment:
```suggestion
    with open(op.join(d, 'bad.txt'), mode='rb') as f:
```

Reply:
Fixed by changing the newline write mode of the test file instead.